README.md

```diff
@@ -3,10 +3,12 @@ ODS-CI is a framework to test [Red Hat Open Data Science](https://www.redhat.com
 and its upstream project, [Open Data Hub](https://opendatahub.io/).
 
 # Requirements
-  Linux distribution that supports Selenium automation of a chromium web browser using [ChromeDriver](https://chromedriver.chromium.org)
-  * chromedriver binaries can be downloaded from https://chromedriver.chromium.org/downloads. The chromedriver version must match the installed version of chromium/google-chrome
+  1. Linux distribution that supports Selenium automation of a chromium web browser using [ChromeDriver](https://chromedriver.chromium.org):
+     * chromedriver binaries can be downloaded from https://chromedriver.chromium.org/downloads
+     * the chromedriver version must match the installed version of chromium/google-chrome
```

Comment:
Can you update this to point to the new chromedriver website? https://googlechromelabs.github.io/chrome-for-testing/
I'd also add Firefox/geckodriver compatibility as an option here

Reply:
* ChromeDriver link to downloads updated
* I'm not sure whether I got your comment correctly but I added a Firefox geckodriver to the descriptions too - please review whether it's what you meant and that it's correct.

Reply:
Yep, everything looks good to me, thanks :D